.github/workflows/pr.yaml

```diff
@@ -0,0 +1,79 @@
+on: pull_request
+name: Review
+jobs:
+  'assets-coding-standards-pratchett':
+    name: Check assets coding standards (pratchett)
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: ['10']
```

Comment:
You should update this to node 16 (which is the current LTS release). Please do this for all jobs.

Reply:
Not required but one could also update the docker-compose setup and use that to ensure that the same version is used both in actions and in lokal development.

Reply:
Our assets cannot be built with node 16 (!). We'll fix this in another pull request.